gcp-observability/src/test/java/io/grpc/gcp/observability/GcpObservabilityTest.java

```diff
@@ -18,48 +18,186 @@
 
 import static com.google.common.truth.Truth.assertThat;
 import static org.junit.Assert.fail;
+import static org.mockito.AdditionalAnswers.delegatesTo;
 import static org.mockito.Mockito.mock;
 import static org.mockito.Mockito.verify;
+import static org.mockito.Mockito.when;
 
+import io.grpc.CallOptions;
+import io.grpc.Channel;
+import io.grpc.ClientCall;
+import io.grpc.ClientInterceptor;
+import io.grpc.InternalGlobalInterceptors;
+// import io.grpc.ManagedChannelProvider;
 import io.grpc.ManagedChannelProvider;
+import io.grpc.Metadata;
+import io.grpc.MethodDescriptor;
+import io.grpc.ServerCall;
+import io.grpc.ServerCallHandler;
+import io.grpc.ServerInterceptor;
+// import io.grpc.ServerProvider;
```

Comment:
another commented import line? Any reason to leave them here?

Reply:
This was an oversight. Removed both.